internal/core/data/router.go

```diff
@@ -774,7 +821,7 @@ func readingHandler(
 
 		pkg.Encode(r, w, loggingClient)
 	case http.MethodPost:
-		reading, err := decodeReading(r.Body, loggingClient, dbClient)
+		r, err := decodeReading(r.Body, loggingClient, dbClient)
```

Comment:
Can we undo the variable name change from `reading`?

Reply:
Change was intentional; reading is a package name.  What I didn't realize was r was already in use.

Reply:
My apologies, I thought I put that into the original comment and instead left that unclear one.